ui/Screen/Org/Projects.svelte

```diff
@@ -2,54 +2,81 @@
-  const registerMember = () => push(path.memberRegistration(params.id));
+  const select = (event) => {
+    const orgProject = event.detail;
+    if (orgProject.maybeProject) {
+      push(path.projectSource(orgProject.maybeProject.id));
+    }
+  };
+
+  $: fetchProjectList({ id: params.id });
+
+  const statsProps = (stats) => {
+    return [
+      { icon: Icon.Commit, count: stats.commits },
+      { icon: Icon.Branch, count: stats.branches },
+      { icon: Icon.Member, count: stats.contributors },
+    ];
+  };
+
+  const projectCardProps = (orgProject) => {
+    if (orgProject.maybeProject) {
+      return {
+        title: orgProject.name,
+        description: orgProject.maybeProject.metadata.description,
+        showRegisteredBadge: true,
+      };
+    } else {
+      return {
+        title: orgProject.name,
+        showRegisteredBadge: true,
+      };
+    }
+  };
 </script>
 
-<style>
-  .wrapper {
-    margin-top: 156px;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-  }
-
-  .content {
-    width: 420px;
-  }
-
-  .buttons {
-    display: flex;
-    justify-content: center;
-  }
-</style>
-
-<div class="wrapper" data-cy="projects">
-  <div class="content">
-    <Placeholder style="width: 100%; height: 217px; margin-bottom: 24px;" />
-    <Text style="margin-bottom: 24px;">
-      There's nothing here yet. Get started by creating your first project or
-      adding a member to your organization.
-    </Text>
-    <div class="buttons">
-      <Button
-        variant="vanilla"
-        style="margin-right: 25px;"
-        on:click={registerProject}>
-        Register a project
-      </Button>
-      <Button
-        variant="vanilla"
-        on:click={registerMember}
-        dataCy="add-member-button">
-        Add a member
-      </Button>
-    </div>
+<Remote store={orgsStore} let:data={orgProjects}>
+  {#if orgProjects.length > 0}
+    <List items={orgProjects} on:select={select} let:item={orgProject}>
+      {#if orgProject.maybeProject}
+        <Flex style="flex: 1">
+          <div slot="left">
+            <ProjectCard {...projectCardProps(orgProject)} />
+          </div>
+
+          <div slot="right" style="display: flex; align-items: center;">
+            <Stats stats={statsProps(orgProject.maybeProject.stats)} />
+            <AdditionalActionsDropdown headerTitle={orgProject.name} />
+          </div>
+        </Flex>
+      {:else}
+        <!-- TODO(julien): what should the registered but no coco metadata
+        state look like visually? -->
+        <Flex style="flex: 1">
+          <div slot="left">
+            <ProjectCard {...projectCardProps(orgProject)} />
+          </div>
+          <div slot="right" style="display: flex; align-items: center;">
+            <AdditionalActionsDropdown headerTitle={orgProject.name} />
+          </div>
+        </Flex>
+      {/if}
+    </List>
+  {:else}{push(path.orgOnboard(params.id))}{/if}
```

Comment:
Having routing invocations which are not triggered by user interaction in the markup is burying crucial behaviour in an unexpected place. Can we solve this differently?

Reply:
Can do, got the inspiration from you: https://github.com/radicle-dev/radicle-upstream/pull/310/files#diff-1f4e722feb2be79d26b81358601f6f3cR21 ; P